docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,257 @@
+
+- Form a Thread network using the OpenThread RCP device and configure IPv6 packet routing to the network.
+- Configure a Wi-Fi hotspot using a spare Wi-Fi card on your PC.
+
+### Forming Thread network
+
+To form a Thread network, complete the following steps:
+
+1. Create an IPv6 network for the OpenThread Border Router (OTBR) container in Docker:
+
+        $ docker network create --ipv6 --subnet 2001:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
+
+2. Start the OTBR container using the command below with *rcp-dk* in the last line replaced with the device node name of the nRF52840 DK that is running the RCP firmware (for example, */dev/ttyACM0*):
+
+        $ docker run -it --rm --privileged --network otbr -p 8080:80 \
+                --sysctl "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" \
+                --volume rcp-dk:/dev/ttyACM0 openthread/otbr --radio-url spinel+hdlc+uart:///dev/ttyACM0
+
+3. Open the `http://localhost:8080/` address in a web browser.
+4. Click **Form** on the left menu.
+   The network forming creator window appears.
+5. Make sure that the On-Mesh Prefix is set to `fd11:22::`.
+   This value is used later to configure the IPv6 packet routing.
+6. Click the **Form** button at the bottom of the window to form a new Thread network using the default settings.
+7. Run the following command to learn which IPv6 address was assigned to the OTBR container in Docker.
+
+        $ docker network inspect otbr | grep IPv6Address
+
+   After checking the output, the address will most likely be `2001:db8:1::2`.
+8. To ensure that packets targeting Thread network nodes are routed through the OTBR container in Docker, run the following command, with *2001:db8:1::2* replaced with the actual address obtained in the previous step:
+
+        $ sudo ip -6 route add fd11:22::/64 dev otbr0 via 2001:db8:1::2
+
+### Configuring Wi-Fi hotspot
+
+To configure a Wi-Fi hotspot using a spare Wi-Fi card on your PC, complete the following steps:
+
+1. Open the Ubuntu settings widget by running the following command:
+
+        $ gnome-control-center
+
+2. Go to the Wi-Fi settings.
+3. Click the three-dot icon at the title bar and select the **Turn On Wi-Fi Hotspot...** option.
+4. Enter your network name and password and click **Turn On**.
+5. Run the following command to assign a well-known IPv6 address to the hotspot interface:
+
+        $ nmcli connection modify Hotspot ipv6.addresses 2001:db8:2::1/64
+
+6. Run the following command to install Routing Advertisement Daemon (radvd), which enables the IPv6 auto-configuration of devices that connect to the hotspot:
+
+        $ sudo apt install radvd
+
+7. Learn the hotspot interface name by running the following command:
+
+        $ nmcli connection show Hotspot | grep interface-name
+
+8. Add the following code into `/etc/radvd.conf`, with *wlo1* replaced with the hotspot interface name obtained in the previous step:
```

Comment:
Is _code_ the right term if it's just a configuration file?

Reply:
You can also write "lines".